operator/src/test/java/org/keycloak/operator/testsuite/apiserver/CRDTest.java

```diff
@@ -33,12 +33,13 @@
 
 import com.fasterxml.jackson.databind.ObjectMapper;
 import org.keycloak.operator.crds.v2alpha1.realmimport.KeycloakRealmImportBuilder;
+import org.keycloak.operator.testsuite.integration.BaseOperatorTest;
 import org.keycloak.operator.testsuite.utils.K8sUtils;
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 
-@KubernetesTest
+@EnableKubeAPIServer
```

Comment:
I wonder if this really saves so much time. We're not starting the Operator in this test, K8s is running already for other tests. Do we know what difference does it make in comparison to interacting with the real K8s API server?

Reply:
This isn't about saving time - yet. While k8s is currently running for the other tests currently, this style of test in the future provides not only a better dev experience - no minikube or kubernetes setup - but it forces us to focus on testing without resorting to the full server with a better "mock" experience that using the built-in fabric8 mock logic.

Reply:
Ok, thanks for clarifying. In this specific case it's IMHO ok to take the "mocking" approach but we'll need to carefully decide each test that should use this approach.